### ISSUE TITLE
--no-interact doesn't work on phpbrew each

### ISSUE THREAD

**Issue by redshark1802** [phpbrew/phpbrew#760]
### Output

```
vagrant@vagrant:/opt/phpbrew$ phpbrew each 'ext install +dev' --no-interact
Run ext install +dev for php-[7.0]. Proceed? [Yes/no]> 

vagrant@vagrant:/opt/phpbrew$ phpbrew each --no-interact 'ext install +dev' 
GetOptionKit\Exception\InvalidOptionException: Invalid option: --no-interact
Trace:

    0) GetOptionKit\ContinuousOptionParser->parse(['/usr/bin/phpbrew', 'each', '--no-interact', 'ext install +dev'])
    1) GetOptionKit\ContinuousOptionParser->continueParse()
    2) CLIFramework\Application->run(['/usr/bin/phpbrew', 'each', '--no-interact', 'ext install +dev'])
    3) PhpBrew\Console->runWithTry(['/usr/bin/phpbrew', 'each', '--no-interact', 'ext install +dev'])
    4) require('phar:///usr/bin/phpbrew/scripts/phpbrew-emb.php')

```
### Expected Result

The command should continue by using the default value `Yes`

**Comment by c9s** [comment]
--no-interact should be put after "phpbrew"

Simon @redshark1802 <notifications@github.com> 於 2016年6月14日 星期二寫道：

**Comment by redshark1802** [comment]
Doesn't work
``
vagrant@vagrant:~$ phpbrew --no-interact each 'ext install +dev'
Run ext install +dev for php-[7.0, 5.6, 5.5, 5.4]. Proceed? [Yes/no]>
``

**Comment by c9s** [comment]
Oh i guess its because "each" is a bash function, not a php command.

Simon @redshark1802 <notifications@github.com> 於 2016年6月14日 星期二寫道：

**Comment by raphaelstolt** [comment]
If this would work it would enable a local, multi version testing like Travis CI. For context see https://github.com/phpbrew/phpbrew/issues/800.

**Comment by morozov** [comment]
Fixed by #888.
